checklist-service/test/services/checklists/list-collaborators.test.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
```

Comment:
I understood that we were omitting list collaborators for now. Should this test be removed?